### ISSUE TITLE
dataFormat with div no longer allows for selection

### ISSUE THREAD

**Issue by wgorman** [AllenFang/react-bootstrap-table#994]
In version 2.9.2, due to the following change:

https://github.com/AllenFang/react-bootstrap-table/commit/eff49cfd90ba7ebbd343c2da921a041c5e310853

my dataFormat which includes a div no longer generates select row events.

My dataFormat looks something like this:

```
var nameFormat = function(cell, row) {
      return (
        <div>
          <div>{cell}</div>
          <div>{row.addlText}</div>
        </div>);
    };
```

Thanks!

Will

**Comment by AllenFang** [comment]
@wgorman, ok, I'll check this out. thanks your feedback 👍

**Comment by AllenFang** [comment]
BTW, could you please show me the configuration for `selectRow` and did you enable cedd editing?

**Comment by wgorman** [comment]
Here is my selectRow config.  I'm not using cell editing for this table.  Note that this works fine in version 2.9.1.
```
{
      mode: "radio",
      clickToSelect: true,
      bgColor: "#EAF3FF",
      onSelect: rowSelect,
      hideSelectColumn: true,
      selected: this.getSelectedKeys()
    }
```

**Comment by AllenFang** [comment]
@wgorman, thanks 👍 I'll fix it as soon as possible

**Comment by AllenFang** [comment]
Fixed on `v2.11.0`, check it out and let me know if the problem still remain.
